crates/blockifier/src/execution/native_syscall_handler.rs

```diff
@@ -25,7 +25,7 @@ use starknet_types_core::felt::Felt;
 use super::execution_utils::max_fee_for_execution_info;
 use super::sierra_utils::{
     allocate_point, big4int_to_u256, calculate_resource_bounds, contract_address_to_felt,
-    encode_str_as_felts, felt_to_starkfelt, starkfelt_to_felt, u256_to_biguint,
+    encode_str_as_felts, felt_native_to_stark_felt, stark_felt_to_felt_native, u256_to_biguint,
```

Comment:
This is a final nitpick, feel free to dismiss it. I think the minor change to `native_felt_to_stark_felt` it might be better since I find it more consistent. So the describing value goes in front just as with stark felt